commons-jcs-jcache/src/main/java/org/apache/commons/jcs/jcache/JCSCachingManager.java

```diff
@@ -100,10 +101,10 @@ public JCSCachingManager(final CachingProvider provider, final URI uri, final Cl
     private Properties readConfig(final URI uri, final ClassLoader loader, final Properties properties) {
         final Properties props = new Properties();
         try {
```

Comment:
I don't know the final intention of the URI parameter but uri.toURL() returns an exception when the URI starts with "jcs" because it is not a valid protocol.
So I tried to read the configuration form the URI as a relative resource as I think it was expected, allowing JCSCachingManager get configured.
See more comments in the issue at jira https://issues.apache.org/jira/browse/JCS-193